Masquerade4997Code/src/main/java/org/firstinspires/ftc/teamcode/Robots/MarkOne/Autonomous/Blue/BlueBuildBridgeAuto.java

```diff
@@ -17,13 +17,11 @@
     @Override
     public void runLinearOpMode() throws InterruptedException{
         robot.init(hardwareMap);
-
-        robot.driveTrain.setClosedLoop(true);
         robot.initializeAutonomous();
+        robot.driveTrain.setClosedLoop(true);
         robot.lift.setClosedLoop(true);
         robot.lift.setKp(0.001);
```

Comment:
Do this type of stuff in the init method. Unless this is opmode specific, which it doesn't look like it is.